docs/framework/wcf/feature-details/troubleshooting-correlation.md

```diff
@@ -198,13 +196,13 @@ public class AddItemMessage
 
  This message contract is used by a <xref:System.ServiceModel.Activities.Receive> activity in a workflow. The `CartId` in the header of the message is used to correlate the message to the correct instance. If the XPath query that retrieves the `CartId` is created using the correlation dialogs in the workflow designer, the following incorrect XPath query is generated.
 
-```
+```xaml
```

Comment:
This and line 207 don't look like xaml to me.

Reply:
I removed the language ID's on these because I don't know what to put.